src/v/cluster/cluster_link/frontend.h

```diff
@@ -104,7 +104,7 @@ class frontend : public ss::peering_sharded_service<frontend> {
     table* _table;
     ss::abort_source* _as;
     cluster::controller_stm* _controller;
-    features::feature_table* _features;
+    [[maybe_unused]] features::feature_table* _features;
```

Comment:
@michael-redpanda I marked this maybe_unused because I didn't know if you had plans to use this later since I know this is all getting changes rapidly.